html/files/knowledge_panel_test.json

```diff
@@ -0,0 +1,92 @@
+        "grade": "d",
+        "icon_url": "https://static.openfoodfacts.org/images/attributes/ecoscore-d.svg",
+        "elements": [
+            {
+                "element_type": "text",
+                "element": {
+                    "text_type": "summary",
+                    "html": "The <a href=\"https://world.openfoodfacts.org/eco-score-the-environmental-impact-of-food-products\">Eco-Score</a> is an experimental score that summarizes the environmental impacts of food products."    
+                }
+            },
+            {
+                "element_type": "text",
+                "element": {
+                    "text_type": "note",
+                    "html": "The Eco-Score was initially developped for France and it is being extended to other European countries. The Eco-Score formula is subject to change as it is regularly improved to make it more precise and better suited to each country."    
+                }
+            },
+            {
+                "element_type": "panel",
+                "element": {
+                    "panel_id": "ecoscore_lca"
+                }  
+            },
+            {
+                "element_type": "text",
+                "element": {
+                    "text_type": "warning",
+                    "html": "<strong>Warning: some information necessary to calculate the Eco-Score with precision is not provided (see the details of the calculation below). You can edit the product to add the missing information.<br><br>If you are the manufacturer of this product, you can send us the information with our <a href=\"https://world.pro.openfoodfacts.org\">free platform for producers</a>."
+                }
+            }
+        ]
+    },
+    "ecoscore_lca" : {
+        "type": "ecoscore_lca",
+        "level": "info",
+        "title": "Lifecyle Analysis (LCA) for the product category",
+        "subtitle": "Score: 40/100",
+        "grade": "b",
+        "elements": [
+            {
+                "element_type": "text",
+                "element": {
+                    "text_type": "summary",
+                    "html": "<p>Agribalyse category: <a href=\"https://www.agribalyse.fr/app/aliments/31032\" title=\"Exact match with the product category\">Chocolate spread with hazelnuts</a></p><ul><li>PEF environmental score: 0.74 (the lower the score, the lower the impact)</li><li>- including impact on climate change: 9.87 kg CO2 eq/kg of product</li></ul>"
+                }
+            },
+            {
+                "element_type": "table_2_columns",
+                "element": {
+                    "table_id": "ecoscore_lca_impacts_by_stages",
+                    "table_type": "percents",
+                    "title": "Details of the impacts by stages of the life cycle",
+                    "headers": ["Steps", "Impact"],
+                    "rows": [
+                        {"id": "agriculture", "title": "Agriculture", "value": "82.7"},
```

Comment:
what's the use case of this id ?

Reply:
It's to allow clients to decide to display specific rows in a specific way if they choose to.

Reply:
do you have an example? I'm still not able to fully imagine the use case in my head, how will the client choose to display these differently, if they would use different icons etc, should we also pass the icons?

Anyways, I was kind of hoping we would start the first version with the basic cases, and slowly move to complicated cases like these. My principle is to avoid coding specialized use cases unless absolutely needed.

Reply:
Then you can ignore the "id" field for now, and we will see if we need it later. It's a good test case anyway: we will extend the knowledge panels, and clients should not break if an extra field they don't know about is added.

Reply:
Cool, sounds good.